modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -108,6 +211,69 @@ NOTE: There is an open https://github.com/tinymce/tinymce/discussions/7487[reque
 If a community-maintained version of the `paste` plugin becomes available, we will link to it from the xref:migration-from-5x.adoc#plugins-paste[_Migration Guide_].
 
 
+=== Real-time Collaboration
+
+The following improvements, changes, and fixes were made to Real-time Collaboration.
+
+Three additional options are now supported when Real-time Collaboration is active:
+
+* `valid_elements`
+* `valid_children`
+* `extended_valid_elements`
+
+Real-time Collaboration now supports the `autocompleter()` API.
+
+//xref:autocompleter.adoc[`autocompleter()`] ?
+
+Real-time Collaboration now supports the `inline_boundaries` option.
+
+// xref:partial$configuration/inline-boundaries.adoc[`inline_boundaries`] ?
+
+When provider values are incorrect, the returned error messages are now less opaque.
+
+For example, the error message `'key' was not a string` is now the error message `Expected field 'key'`.
+
+Previously, if a {productname} editor instance with Real-time Collaboration loaded contained a Horizontal Rule (ie an `+<hr>+` element), attempting to move the insertion point over the Horizontal Rule using arrow keys threw an exception: `Uncaught Error: Unsupported node type: [object DocumentType]`.
```

Comment:
```suggestion
Previously, if a {productname} editor instance with Real-time Collaboration loaded and contained a Horizontal Rule (ie an `+<hr>+` element), attempting to move the insertion point over the Horizontal Rule using arrow keys threw an exception: `Uncaught Error: Unsupported node type: [object DocumentType]`.
```

Reply:
To quote my git commit comment:

Conjunction — also — added to sentence regarding HR bug fix. A different conjunction — and — was suggested in the PR but, on reflection, the potential for reader confusion is obviated more effectively by adding ‘also’ than by adding ‘and’.